File: backend/app/api/routes/ping.py
import json

from fastapi import APIRouter, Depends, HTTPException
from app.services.event import send_event

router = APIRouter()

@router.get("")
async def check_ping():
    try:
        res = send_event('ping', {})
        return res.json()

    except Exception as e:
        print(e)
        raise HTTPException(status_code=500, detail="Error")

File: backend/routes/api/ping.js
const router = require("express").Router();
const asyncHandler = require("express-async-handler");
const auth = require("../auth");
const { sendEvent } = require("../../lib/event");

router.get("/",
  auth.optional,
  asyncHandler(async (req, res) => {

    try {
      const result = await sendEvent('ping')
      return res.json(result);
    } catch (e) {
      console.error(e)
      return res.sendStatus(500);
    }
  }));

module.exports = router;
